docs/latest/modules/en/pages/k8s-suse-rancher-prime.adoc

```diff
@@ -105,6 +105,10 @@ The minimum requirements do not include spare CPU/Memory capacity to ensure smoo
 
 SUSE Observability uses persistent volume claims for the services that need to store data. The default storage class for the cluster will be used for all services unless this is overridden by values specified on the command line or in a `values.yaml` file. All services come with a pre-configured volume size that should be good to get you started, but can be customized later using variables as required.
 
+[NOTE]
+====
+SUSE Observability requires the underlying storage to be based on flash memory (SSD) or similar in performance.
```

Comment:
Only mentioning SSD still leaves open a lot of options, but it is clear enough in most cases I think given we don't have a hard number of IOPS.